java/src/gov/usgs/consumerclient/ConsumerClient.java

```diff
@@ -135,10 +138,15 @@ public static void main(String[] args) {
 		StringBuffer configBuffer = new StringBuffer();
 		try {
 			configReader = new BufferedReader(new FileReader(configFile));
-			String text = null;
-
-			while ((text = configReader.readLine()) != null) {
-				configBuffer.append(text).append("\n");
+			String line = null;
+
+			while ((line = configReader.readLine()) != null) {
```

Comment:
generic config file parsing code take 4...

Doesn't seem like stripCommentsFromLine() ever returns null, so why are you checking for null.
Really seems like this code could just be in one place, a magical place, that we could call a library.....
OK... OK... </rant>

Reply:
An abundance of caution?